Source/Laser/LaserProfiles.H

```diff
@@ -0,0 +1,156 @@
+#ifndef WARPX_LaserProfiles_H_
+#define WARPX_LaserProfiles_H_
+
+#include <AMReX_REAL.H>
+#include <WarpXParser.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_Vector.H>
+
+#include <WarpXParser.H>
+
+#include <map>
+#include <string>
+#include <memory>
+#include <functional>
+#include <limits>
+
+namespace WarpXLaserProfiles {
+
+//_________________________________________Laser parameters shared among classes
+struct CommonLaserParameters
+{
+    amrex::Real wavelength;
+    amrex::Real e_max;
```

Comment:
Very important for talk to experimentalists:
```suggestion
    amrex::Real e_max;  //! maximum electric field at peak
```